bld/CLMBuildNamelist.pm

```diff
@@ -60,6 +60,7 @@ SYNOPSIS
      Create the namelist for CLM
 REQUIRED OPTIONS
      -cimeroot "directory"    Path to cime directory
+     -landroot "directory"    Path to the land models parent directory, ie <>/ctsm
```

Comment:
We probably don't need to send landroot in, as we should be able to get it from information already in CLMBuildNamelist.pm